addons/core/lib/feature-parser.js

```diff
@@ -15,7 +15,7 @@ class FeatureParser extends Filter {
     const contentEscaped = content.replace(/`/g, '\\`');
 
     const module = `
-      import TestDeclarator from '@ember-bdd/core/test-support/test-declarator';
+      import TestDeclarator from '@ember-bdd/core/test-declarator';
```

Comment:
Intentionally not importing from `@ember-bdd/core` (which also does not reexport this), as I *think* this should be a private API only used by the feature parser, right?

Reply:
OK!